GetRepoJSON/getRepoJSON.py

```diff
@@ -0,0 +1,52 @@
+# repos.py
+
+# A demo script that demonstrates api calls /user
+
+
+
+from time import sleep
+def delayserver():
+    sleep(0.01)
+    return
+
+
+import sys
+sys.path.append('..')
+from allspice_proto.allspice_proto import AllSpice_Proto
+# from allspice.allspice import AllSpice
+# allspice = AllSpice()
+allspice = AllSpice_Proto()
+# delayserver()
+allspice.start()
+# allspice.hub.logEveryEndpoint(True)
+
+delayserver()
+
+from pygitea_mod.gitea.apiobject import Repository
+
+
+# -----------------------------------------------------------------------------
+allspice.infoheader("Test GET /version")
+# test GET server version
+versionResponse = ""
```

Comment:
Variables should be snake case.

```suggestion
version_response = ""
```

Reply:
I converted all the local variables to snake case.